src/nemos/basis.py

```diff
@@ -1471,14 +1573,24 @@ def __init__(
         mode="eval",
         width: float = 2.0,
         window_size: Optional[int] = None,
+        bounds: Optional[Tuple[float, float]] = None,
         **kwargs,
     ) -> None:
         super().__init__(
-            n_basis_funcs, *args, mode=mode, window_size=window_size, **kwargs
+            n_basis_funcs,
+            *args,
+            mode=mode,
+            window_size=window_size,
+            bounds=bounds,
+            **kwargs,
         )
         self._n_input_dimensionality = 1
         self._check_width(width)
         self._width = width
+        # if linear raised cosine are initialized
+        # this flag is always true, the samples
+        # must be rescaled to 0 and 1.
```

Comment:
```suggestion
        # for these linear raised-cosine basis functions,
        # the samples must be rescaled to 0 and 1.
```

the "if" in previous wording is confusing